Makefile

```diff
@@ -1,6 +1,7 @@
 .PHONY: compile release test
 plugins=*
 GRUNT=node_modules/.bin/grunt
+CURRENT_VERSION := $(shell sed -n '/"version":/{s/.*"version": "\([^"]*\)".*/\1/p;q}' package.json)
```

Comment:
Here is how this sed expression might look like in a more human-readable form:
```
/"version":/ {                         # if we found a line that contains this pattern
    s/.*"version": "\([^"]*\)".*/\1/p; # extract a version out of this line and print it (otherwise it won't because we passed `-n` option
    q                                  # stop script execution: we don't want to check other lines as we need only the first occurence
}
```

Reply:
P.S. Hm... but this only work with GNU sed. Should we care about it?

Reply:
Not sure we need to care, as this isn't something that is run often, and we usually have control over the environments.  If we need to do something more robust later we can.

Good Work